src/main/java/org/jvnet/hudson/test/junit/jupiter/JenkinsRuleResolverTest.java

```diff
@@ -0,0 +1,20 @@
+package org.jvnet.hudson.test.junit.jupiter;
```

Comment:
You seem to have put tests in `src/main/` by accident?

Reply:
Wow sorry. I don't know how it ended here.